### PR TITLE
simplify and clean tydiqa templates

### PR DESCRIPTION
**Pull request by VictorSanh**
unfortunately the two splits are not the same...

### CODE REVIEW

File: promptsource/templates/tydiqa/primary_task/templates.yaml [hunk 1]
```diff
@@ -1,127 +1,174 @@
...
       \    {% endif %}\n{% endif %}"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: false
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
       original_task: true
```
Comment (craffel) [inline promptsource/templates/tydiqa/primary_task/templates.yaml:139]: The prompt `open_domain_qa_without_choices` has `original_task: false`, but it seems it is no less original-tasky than this one. Should one or the other be changed?

File: promptsource/templates/tydiqa/primary_task/templates.yaml [hunk 2]
```diff
@@ -1,127 +1,174 @@
...
       _do_eval: false
       _do_train: false
-      choices_in_prompt: null
-      metrics: []
-      original_task: true
-    name: binary_question_without_context_1
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: false
```
Comment (craffel) [inline promptsource/templates/tydiqa/primary_task/templates.yaml:156]: Same here about `original_task`

File: promptsource/templates/tydiqa/primary_task/templates.yaml [hunk 3]
```diff
@@ -1,127 +1,174 @@
...
+      \ == \"YES\" or annotations.yes_no_answer[0] == \"NO\" %} \nBased on the following\
+      \ text snippet, {{question_text}} \n{{document_plaintext}}\n||| \n{{annotations.yes_no_answer[0]\
+      \ | capitalize}}\n    {% endif %}\n{% endif %}"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: false
-      choices_in_prompt: null
+      choices_in_prompt: false
       metrics: []
       original_task: true
```
Comment (craffel) [inline promptsource/templates/tydiqa/primary_task/templates.yaml:172]: Same here about `original_task`

**Review by craffel — COMMENTED** [review]
Some questions about `original_task`. Also, can `answer_choices_key` be something like `{{annotations.yes_no_answer[0] | capitalize}}` or would that not work? Otherwise LGTM, please merge away.

**Comment by VictorSanh** [comment]
> Some questions about `original_task`. 

My rationale for `original_task`: if it uses the text of the article then it's original task, otherwise it's closer to opendomain qa and thus didn't tag it as original task. 

> Also, can `answer_choices_key` be something like `{{annotations.yes_no_answer[0] | capitalize}}` or would that not work? 

`{{annotations.yes_no_answer[0] | capitalize}}` would definitely work but I wouldn't know whether it corresponds to Yes or No, and thus don't know which other option to list.

> Otherwise LGTM, please merge away

Merging now

**Pull request merged**